firebase-fetcher/build.gradle

```diff
@@ -17,10 +18,17 @@ android {
     }
 }
 
+repositories {
```

Comment:
i believe that this is unnecessary. Repositories should be defined in root `build.gradle`

Reply:
1625e84cb028ce7b65cc6bd34201660fe27780f8